src/Tools/CoreFx.Tools/AssemblyResolver.cs

```diff
@@ -0,0 +1,109 @@
+// Licensed to the .NET Foundation under one or more agreements.
```

Comment:
I suspect having this in the Tools directory might cause some conflicts with the Tools we create for the toolset. At a minimum I think the entire directory is gitignored so people will have to force add to change anything.